Patches/Androids Expanded/AlienRace/AlienRace_Droids.xml

```diff
@@ -0,0 +1,246 @@
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/statBases/ArmorRating_Sharp</xpath>
+				  <value>
+				    <ArmorRating_Sharp>16</ArmorRating_Sharp>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/statBases/ArmorRating_Blunt</xpath>
+				  <value>
+				    <ArmorRating_Blunt>36</ArmorRating_Blunt>
+				  </value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+				  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="ChjSpacerBattleDroid"]/tools</xpath>
+				  <value>
+				    <tools>
+					  <li Class="CombatExtended.ToolCE">
+						<label>left razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>LeftDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>right razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>RightDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>lower left razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>LeftDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>lower right razor-fist</label>
+						<capacities>
+						  <li>Cut</li>
+						  <li>Stab</li>
+						  <li>Blunt</li>
+						</capacities>
+						<power>27</power>
+						<cooldownTime>1.76</cooldownTime>
+						<linkedBodyPartsGroup>LowerRightDroidHand</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>60</armorPenetrationBlunt>
+						<armorPenetrationSharp>24</armorPenetrationSharp>
+					  </li>
+					  <li Class="CombatExtended.ToolCE">
+						<label>giant body slam</label>
+						<capacities>
+						  <li>Blunt</li>
+						</capacities>
+						<power>65</power>
+						<chanceFactor>0.33</chanceFactor>
+						<cooldownTime>4.17</cooldownTime>
+						<linkedBodyPartsGroup>Torso</linkedBodyPartsGroup>
+						<armorPenetrationBlunt>30</armorPenetrationBlunt>
+					  </li>
+					</tools>
+				  </value>
+				</li>
+				
+			<li Class="PatchOperationAdd">

```

Comment:
I'd remove this part completely. The textures get really weird on the battle droid, and there's really no need for any of them. Its carrying capacity is sufficient already, and it really doesn't need the extra armor of a shield. It just looks silly.